src/connectionmanager.js

```diff
@@ -707,17 +583,12 @@ export default class ConnectionManager {
             // Clone the array
             const credentials = credentialProvider.credentials();
 
-            return Promise.all([getConnectServers(credentials), findServers()]).then(responses => {
+            return Promise.all([findServers()]).then(responses => {
 
-                const connectServers = responses[0];
-                const foundServers = responses[1];
+                const foundServers = responses[0];
 
                 let servers = credentials.Servers.slice(0);
-                mergeServers(credentialProvider, servers, foundServers);
-                mergeServers(credentialProvider, servers, connectServers);
-
-                servers = filterServers(servers, connectServers);
-
+                
```

Comment:
Is there trailing whitespace here or was it removed? I think it was removed, just want to be sure.